core-feature-pack/src/main/resources/content/bin/add-user.ps1

```diff
@@ -0,0 +1,25 @@
+#############################################################################
+#                                                                          ##
+#    WildFly Startup Script for adding users							   ##
+#                                                                          ##
+#############################################################################
+$RESOLVED_JBOSS_HOME = (Get-ChildItem $MyInvocation.MyCommand.Path).Directory.Parent.FullName
+. "$RESOLVED_JBOSS_HOME\bin\common.ps1"
+
+$JAVA_OPTS = @()
+
+# Sample JPDA settings for remote socket debugging
+#$JAVA_OPTS+="-agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=y"
+# Uncomment to override standalone and domain user location
+#$JAVA_OPTS+="-Djboss.server.config.user.dir=$JBOSS_HOME/standalone/configuration"
+#$JAVA_OPTS+="-Djboss.domain.config.user.dir=$JBOSS_HOME/domain/configuration"
+
+
+$PROG_ARGS = Get-Java-Arguments -entryModule "org.jboss.as.domain-add-user" -serverOpts $ARGS -logFileProperties $SCRIPTS_HOME\add-user.properties
```

Comment:
method Get-Java-Arguments takes it as optional parameter, to build the all java arguments and return it as array to be that used to start the java executable.
Current version of add user script requires that we use special properties file (similar for jboss-cli script) so some special log messages can be displayed.

If parameter is not specified it defaults to our default logging.properties